diff --git a/build/esm/platform/node/instrumentation.js b/build/esm/platform/node/instrumentation.js
index f497f63ce4a43686767c28b2c4c520179d46395c..de5d586f5615c72d69ccad861fb1016b4e44da5c 100644
--- a/build/esm/platform/node/instrumentation.js
+++ b/build/esm/platform/node/instrumentation.js
@@ -39,15 +39,19 @@ var __values = (this && this.__values) || function(o) {
     };
     throw new TypeError(s ? "Object is not iterable." : "Symbol.iterator is not defined.");
 };
+
 import * as path from 'path';
-import { types as utilTypes } from 'util';
-import { satisfies } from 'semver';
-import { wrap, unwrap } from 'shimmer';
+
+import { unwrap, wrap } from 'shimmer';
+
+import { Hook } from 'require-in-the-middle';
+import ImportInTheMiddle from 'import-in-the-middle';
 import { InstrumentationAbstract } from '../../instrumentation';
 import { RequireInTheMiddleSingleton, } from './RequireInTheMiddleSingleton';
-import * as ImportInTheMiddle from 'import-in-the-middle';
 import { diag } from '@opentelemetry/api';
-import { Hook } from 'require-in-the-middle';
+import { satisfies } from 'semver';
+import { types as utilTypes } from 'util';
+
 /**
  * Base abstract class for instrumenting node plugins
  */
