diff --git a/build/esm/platform/node/util.js b/build/esm/platform/node/util.js
index 03b6e97b6ec10b8d2df0689250e224ae54beb39d..62bde571c5112311e68b6bf405ea1d6f5525df9b 100644
--- a/build/esm/platform/node/util.js
+++ b/build/esm/platform/node/util.js
@@ -9,6 +9,9 @@ var __assign = (this && this.__assign) || function () {
     };
     return __assign.apply(this, arguments);
 };
+
+import * as http from 'http';
+import * as https from 'https';
 /*
  * Copyright The OpenTelemetry Authors
  *
@@ -25,15 +28,16 @@ var __assign = (this && this.__assign) || function () {
  * limitations under the License.
  */
 import * as url from 'url';
-import * as http from 'http';
-import * as https from 'https';
 import * as zlib from 'zlib';
+
+import { DEFAULT_EXPORT_BACKOFF_MULTIPLIER, DEFAULT_EXPORT_INITIAL_BACKOFF, DEFAULT_EXPORT_MAX_ATTEMPTS, DEFAULT_EXPORT_MAX_BACKOFF, isExportRetryable, parseRetryAfterToMills, } from '../../util';
+
+import { CompressionAlgorithm } from './types';
+import { OTLPExporterError } from '../../types';
 import { Readable } from 'stream';
 import { diag } from '@opentelemetry/api';
-import { CompressionAlgorithm } from './types';
 import { getEnv } from '@opentelemetry/core';
-import { OTLPExporterError } from '../../types';
-import { DEFAULT_EXPORT_MAX_ATTEMPTS, DEFAULT_EXPORT_INITIAL_BACKOFF, DEFAULT_EXPORT_BACKOFF_MULTIPLIER, DEFAULT_EXPORT_MAX_BACKOFF, isExportRetryable, parseRetryAfterToMills, } from '../../util';
+
 /**
  * Sends data using http
  * @param collector
@@ -52,11 +56,12 @@ export function sendWithHttp(collector, data, contentType, onSuccess, onError) {
     var exporterTimer = setTimeout(function () {
         clearTimeout(retryTimer);
         reqIsDestroyed = true;
-        if (req.destroyed) {
+        
+        if (req && req.destroyed) {
             var err = new OTLPExporterError('Request Timeout');
             onError(err);
         }
-        else {
+        else if (req) {
             // req.abort() was deprecated since v14
             nodeVersion >= 14 ? req.destroy() : req.abort();
         }
diff --git a/build/esnext/platform/node/util.js b/build/esnext/platform/node/util.js
index 2970a14f64b355f8f98772c3426c78e4eadad623..967f11c788bfe641e5f30057a648be99440acf82 100644
--- a/build/esnext/platform/node/util.js
+++ b/build/esnext/platform/node/util.js
@@ -1,3 +1,5 @@
+import * as http from 'http';
+import * as https from 'https';
 /*
  * Copyright The OpenTelemetry Authors
  *
@@ -14,15 +16,16 @@
  * limitations under the License.
  */
 import * as url from 'url';
-import * as http from 'http';
-import * as https from 'https';
 import * as zlib from 'zlib';
+
+import { DEFAULT_EXPORT_BACKOFF_MULTIPLIER, DEFAULT_EXPORT_INITIAL_BACKOFF, DEFAULT_EXPORT_MAX_ATTEMPTS, DEFAULT_EXPORT_MAX_BACKOFF, isExportRetryable, parseRetryAfterToMills, } from '../../util';
+
+import { CompressionAlgorithm } from './types';
+import { OTLPExporterError } from '../../types';
 import { Readable } from 'stream';
 import { diag } from '@opentelemetry/api';
-import { CompressionAlgorithm } from './types';
 import { getEnv } from '@opentelemetry/core';
-import { OTLPExporterError } from '../../types';
-import { DEFAULT_EXPORT_MAX_ATTEMPTS, DEFAULT_EXPORT_INITIAL_BACKOFF, DEFAULT_EXPORT_BACKOFF_MULTIPLIER, DEFAULT_EXPORT_MAX_BACKOFF, isExportRetryable, parseRetryAfterToMills, } from '../../util';
+
 /**
  * Sends data using http
  * @param collector
@@ -41,11 +44,12 @@ export function sendWithHttp(collector, data, contentType, onSuccess, onError) {
     const exporterTimer = setTimeout(() => {
         clearTimeout(retryTimer);
         reqIsDestroyed = true;
-        if (req.destroyed) {
+        
+        if (req && req.destroyed) {
             const err = new OTLPExporterError('Request Timeout');
             onError(err);
         }
-        else {
+        else if (req) {
             // req.abort() was deprecated since v14
             nodeVersion >= 14 ? req.destroy() : req.abort();
         }
diff --git a/build/src/platform/node/util.js b/build/src/platform/node/util.js
index 69ac2e2695a922f505de56b59362a5cfb7d0096e..252a43f7c8dde168eeee3b6cd5a3217678c84fa6 100644
--- a/build/src/platform/node/util.js
+++ b/build/src/platform/node/util.js
@@ -44,11 +44,12 @@ function sendWithHttp(collector, data, contentType, onSuccess, onError) {
     const exporterTimer = setTimeout(() => {
         clearTimeout(retryTimer);
         reqIsDestroyed = true;
-        if (req.destroyed) {
+        
+        if (req && req.destroyed) {
             const err = new types_2.OTLPExporterError('Request Timeout');
             onError(err);
         }
-        else {
+        else if (req) {
             // req.abort() was deprecated since v14
             nodeVersion >= 14 ? req.destroy() : req.abort();
         }
