diff --git a/index.js b/index.js
index 33778b5539b7fcd7a1e99474a4ecb1745fdfe508..2132aac8d60c192dce1a94c0e4b14fad1e569351 100644
--- a/index.js
+++ b/index.js
@@ -9,7 +9,7 @@ module.exports = inquire;
  */
 function inquire(moduleName) {
     try {
-        var mod = eval("quire".replace(/^/,"re"))(moduleName); // eslint-disable-line no-eval
+        var mod = require(moduleName); // eslint-disable-line no-eval
         if (mod && (mod.length || Object.keys(mod).length))
             return mod;
     } catch (e) {} // eslint-disable-line no-empty
