diff --git a/node_modules/react-resize-panel-ts/dist/index.d.ts b/node_modules/react-resize-panel-ts/dist/index.d.ts
index a0f24d9..052e5f6 100644
--- a/node_modules/react-resize-panel-ts/dist/index.d.ts
+++ b/node_modules/react-resize-panel-ts/dist/index.d.ts
@@ -5,6 +5,7 @@ export interface IResizePanelProps {
   borderClass?: string;
   containerClass?: string;
   style?: React.CSSProperties;
+  children?: React.ReactNode;
 }
 
 declare class ResizePanel extends React.Component<IResizePanelProps> {
